autofill/autofill-impl/src/main/AndroidManifest.xml

```diff
@@ -20,6 +20,7 @@
     <application>
         <activity
             android:name="com.duckduckgo.autofill.ui.credential.management.AutofillManagementActivity"
+            android:configChanges="orientation|screenSize"
```

Comment:
should probably copy what we do for `BrowserActivity`?

`android:configChanges="keyboardHidden|orientation|screenSize"`

Reply:
Hmm, Didn't seem necessary though. I don't think keyboardHidden is triggering any restart on the Activity atm. I will not add it for now. Maybe add it later if it will seem needed.

Reply:
IIRC, `keyboardHidden` refers to a hardware keyboard's availability, not the software-based IME. what the best list is, i'm not sure (we're probably also missing `screenLayout` from both for instance).